app/pages/project/stats/index.jsx

```diff
@@ -29,22 +29,26 @@ class ProjectStatsPageController extends React.Component {
 
   getWorkflows(project) {
     const fields = [
+      'active',
       'classifications_count',
       'completeness',
       'configuration',
       'display_name',
       'retired_set_member_subjects_count',
-      'retirement,subjects_count',
+      'retirement,subjects_count'
     ];
     const query = {
-      active: true,
-      fields: fields.join(','),
+      fields: fields.join(',')
     };
     getWorkflowsInOrder(project, query)
       .then((workflows) => {
         const workflowsSetToBeVisible =
-          workflows.filter((workflow) => { 
-            return (!workflow.configuration.stats_hidden ? workflow : null);
+          workflows.filter((workflow) => {
+            let stats_visible = !workflow.configuration.stats_hidden
+            if (!workflow.active) {
+              stats_visible = !(workflow.configuration.stats_hidden === undefined) && stats_visible
```

Comment:
Duplicates the previous block of if logic. I'm not quite following what these comparisons do.